.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
     - DJANGO='Django==1.7.*'
     - DJANGO='Django==1.8.*'
     - DJANGO='Django==1.9.*'
+    - DJANGO='Django==1.10.*'
```

Comment:
@quiqueporta Would you be open to adding 1.11 now? That would be great, especially considering it's an LTS release (might even help to persuade someone to merge this sometime soon… 🤞).